DEVELOPMENT.md

```diff
@@ -105,24 +105,3 @@ To test the rules run
 ```
 npm run tslint:test
```

Comment:
this command does not exist and work anymore

Reply:
As you are the one currently knowing all/most of the commands, could you at some point update the documentation files accordingly?